radio/src/mixer.cpp

```diff
@@ -793,11 +800,7 @@ void evalFlightModeMixes(uint8_t mode, uint8_t tick10ms)
 #endif
         }
       }
-
```

Comment:
2) This code should remain here (so that SLOW could modify weight)

NB: perhaps weight will have to be shifted to avoid precision lost